docs/deployment/getting_started_locally_with_gardenadm.md

```diff
@@ -142,13 +142,13 @@ Use the following command to prepare the `gardenadm` medium-touch scenario:
 make gardenadm-medium-touch-up
 ```
 
-This will first build the needed images and then render the needed manifests for `gardenadm bootstrap` to the [`./example/gardenadm-local/medium-touch`](../../example/gardenadm-local/medium-touch) directory.
+This will first build the needed images and then render the needed manifests for `gardenadm bootstrap` to the [`./example/gardenadm-local/resources/medium-touch`](../../example/gardenadm-local/resources/medium-touch) directory.
 
 Afterward, you can use `go run` to execute `gardenadm` commands on your machine:
 
 ```shell
 $ export IMAGEVECTOR_OVERWRITE=$PWD/example/gardenadm-local/.imagevector-overwrite.yaml
-$ go run ./cmd/gardenadm bootstrap -d ./example/gardenadm-local/medium-touch
+$ go run ./cmd/gardenadm bootstrap -d ./example/gardenadm-local/resources/medium-touch
```

Comment:
The `example/gardenadm-local/medium-touch` directory is no longer used. Can you remove `example/gardenadm-local/medium-touch/.gitignore`?